cmd/konflux-apply/README.md

```diff
@@ -0,0 +1,56 @@
+# Konflux Apply
+
+The Konflux manifests are applied automatically via the [konflux-apply](../../.github/workflows/apply-konflux-manifests.yaml) Github Workflow.
+
+For this workflow, we use a token for Konflux setup as described in [the docs](https://gitlab.cee.redhat.com/konflux/docs/users/-/blob/main/topics/getting-started/getting-access.md#logging-to-the-internal-cluster-with-a-token), with the following service account, role & rolebindings:
+
+1. Service account `gh-action`:
+    ```
+    apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      namespace: ocp-serverless-tenant
+      name: gh-action
+    ```
+2. Role with minimal permissions to apply the Konflux manifests:
+    ```
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: Role
+    metadata:
+      namespace: ocp-serverless-tenant
+      name: gh-action-runner
+    rules:
+    - apiGroups:
+      - appstudio.redhat.com
+      resources:
+      - applications
+      - components
+      - imagerepositories
+      verbs:
+      - get
+      - list
+      - watch
+      - update
+      - patch
+      - create
+      ```
+3. Rolebinding
+    ```
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: RoleBinding
+    metadata:
+      namespace: ocp-serverless-tenant
+      name: gh-action
+    subjects:
+    - kind: ServiceAccount
+      name: gh-action
+      apiGroup: ""
+    roleRef:
+      kind: Role
+      name: gh-action-runner
+      apiGroup: rbac.authorization.k8s.io
+    ```
```

Comment:
did you apply those manually?

Can we place them in a directory ? so that in case we need to restore them we can just apply them?

Reply:
Yes. They're applied already.
Placing them in something like `./cmd/konflux-apply/manifests`? So it's right next to the Readme.md

Reply:
Yes, especially with the revoke + renew flow, it could be very handy

```
$ kubectl delete -f ./cmd/konflux-apply/manifests
$ kubectl apply -f ./cmd/konflux-apply/manifests
$ kubectl create token gh-action ...
```